release.yaml

```diff
@@ -39,7 +39,7 @@ rancher-gke-operator:
 rancher-gke-operator-crd:
   - 104.4.0+up1.9.4-rc.1
 rancher-webhook:
-  - 104.0.4+up0.5.4-rc.1
+  - 103.0.11+up0.4.12
```

Comment:
:thinking: confused, shouldn't that be 0.5.3?

Reply:
Hmm, I will add a commit to fix this.